svthalia/concrexit
# Improve vacancy page layout

Closes #1937 

### Summary
Improved the vacancy layout

### How to test
Steps to test the changes you made:
1. Go to `/career/vacancies`
2. Check if you can see locations and tags
3. Maybe add a location and tags to the vacancies in the admin panel
4. Check again

![Screen Shot 2021-09-20 at 21 41 01](https://user-images.githubusercontent.com/6750407/134064847-be6c6cf6-6793-402d-86c6-4dd3fbaa98fd.png)
